### ISSUE TITLE
trustworthiness or Kary eval metric

### ISSUE THREAD

**Issue by huguesva** [TorchDR/TorchDR#99]
It would be nice to have the trustworthiness as in sklearn :
https://scikit-learn.org/stable/modules/generated/sklearn.manifold.trustworthiness.html

Or the Kary criterion used in :
https://www.sciencedirect.com/science/article/abs/pii/S0925231215003641

**Comment by cedricvincentcuaz** [comment]
Ok on it :rocket:
